### PR TITLE
Package pychrono

### PR DESCRIPTION
**Pull request by davidbrochart**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by davidbrochart** [comment]
Poking CI.

File: recipes/pychrono/conda_buld_config.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+c_compiler: gcc
```
Comment (isuruf) [inline recipes/pychrono/conda_buld_config.yaml]: Please remove this file

File: recipes/pychrono/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
+    - swig 3.0.12
+    - pcre
+    - bzip2
+    - python
+    - zlib
```
Comment (isuruf) [inline recipes/pychrono/meta.yaml]: 4 lines above should be in host.

Comment (SylvainCorlay) [reply recipes/pychrono/meta.yaml]: sorry I missed that.

Comment (isuruf) [reply recipes/pychrono/meta.yaml]: No need to be sorry. Thanks for reviewing.

Comment (SylvainCorlay) [reply recipes/pychrono/meta.yaml]: I will fixup the feedstock as soon as it is created.

**Pull request merged**